helix-lsp/src/lib.rs

```diff
@@ -915,10 +915,17 @@ fn start_client(
         }
 
         // next up, notify<initialized>
-        _client
+        let notification_result = _client
             .notify::<lsp::notification::Initialized>(lsp::InitializedParams {})
-            .await
-            .unwrap();
+            .await;
+
+        if let Err(e) = notification_result {
+            log::error!(
+                "failed to notify language server of its initialization: {}",
```

Comment:
I don't think this includes the name of the LSP to help with a multi-LSP setup, but I'll have to check how that's done.

Reply:
Seems like we aren't handling this at all. Opened #9185